profiles/common/modules/features/nexteuropa_trackedchanges/nexteuropa_trackedchanges.module

```diff
@@ -736,7 +738,7 @@ function nexteuropa_trackedchanges_cron() {
           }
         }
       }
-      if (isset($queue)) {
+      if ($queue !== NULL) {
```

Comment:
FYI According to our coding standard, this should better be if NULL !== $queue

Reply:
Use is_null()

Reply:
iz_true ! 😆